src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +57,16 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Adds {@code error} to {@code errors} if {@code error} is a non-empty string.
+     * @param error A String, which describes an error, can be empty if no error received.
+     * @param errors A List of non-empty String values with {@code error} descriptions,
+     *          which is modified by adding new non-empty {@code error} value, received as first parameter.
+     */
+    public void addNonEmptyError(String error, List<String> errors) {
+        if (!error.isEmpty()) {
+            errors.add(error);
+        }
```

Comment:
@damithc Could you please explane this one - is 'happy path' here suggests empty error message? And if I remove indents wouldn't that be checkstyle violation? Thank you